components/script/dom/resizeobserver.rs

```diff
@@ -288,14 +325,11 @@ impl ResizeObservation {
     }
 
     /// <https://drafts.csswg.org/resize-observer/#dom-resizeobservation-isactive>
-    /// Returning an optional calculated size, instead of a boolean,
-    /// to avoid recalculating the size in the subsequent broadcast.
```

Comment:
I've removed this optimization. Keeping it around would make the code a bit messy because when we initialize the `ResizeObserverEntry` fields then we have one of the three size values (depending on the observation type) and need to calculate the other two. Now that we have incremental layout, I think the cost of querying an element's area (without reflow) is very low.

I'm happy to keep the optimization around if someone objects to this change.